src/main/java/seedu/address/logic/parser/UntagCommandParser.java

```diff
@@ -15,42 +20,66 @@
  */
 public class UntagCommandParser implements Parser<UntagCommand> {
     public static final int MAX_LENGTH = 50;
```

Comment:
`MAX_CHARACTER_LENGTH = 50` already exists in the tag class. Maybe it can be imported

Reply:
Good catch!